string NAME=scan

ObjectDrone d
ObjectLocation l
